massa-execution-exports/src/event_store.rs

```diff
@@ -216,8 +216,8 @@ impl EventStore {
 
     /// get vec of event for given slot range (start included, end excluded)
     /// Get events optionnally filtered by:
-    /// * start slot
-    /// * end slot
+    /// * start slot (default Slot(0,0))
+    /// * end slot   (default Slot(0,0))
```

Comment:
in this function slots are not optional so I'm not sure about the added part here

Reply:
That's really the default values on call the controller to get the outputs events

https://github.com/massalabs/massa/blob/2d61824394cca66a505378be2f046f420b9b19cd/massa-execution-exports/src/controller_traits.rs#L30-L43

I though it was great to repeat that here. I'm wrong?

Reply:
Oh I see your point. I do not have an answer, make it the less surprising possible :)

Reply:
```suggestion
    /// * start slot (considering it's derivated from an optional argument, note that the default will be Slot(0,0))
    /// * end slot  (same comment as above)
```